docs/class-definitions.md

```diff
@@ -42,6 +42,28 @@ Number of milliseconds after a user loses connection or closes their browser win
 
 Default is 120,000ms (2 minutes).
 
+#### cursors
+
+Configure the cursors API.
+
+##### outgoingBatchTime
+
+The interval at which a batch of cursors position is published. This is multiplied by the number of members in the space reduced by 1. (i.e. outgoingBatchTime \* (membersCount - 1)).
+
+Default 100ms.
+
+##### incomingBatchTime
```

Comment:
Should it be `outgoingBatchInterval` and `incomingBatchInterval` instead? Time suggests a static timestamp, no? @dpiatek

Reply:
Agree on 'interval' over 'time'. I'd also suggest using 'inbound' and 'outbound', so: `outboundBatchInterval` and `inboundBatchInterval`. This matches with how we refer to things elsewhere, e.g. [outbound message limits](https://ably.com/docs/general/limits#outbound-message) and 'outgoing' is always an odd one on first look.